src/core/server/saved_objects/service/lib/point_in_time_finder.ts

```diff
@@ -139,7 +139,9 @@ export class PointInTimeFinder<T = unknown, A = unknown>
 
   private async open() {
     try {
-      const { id } = await this.#client.openPointInTimeForType(this.#findOptions.type);
+      const { id } = await this.#client.openPointInTimeForType(this.#findOptions.type, {
+        namespaces: this.#findOptions.namespaces,
+      });
```

Comment:
I don't think propagating `typeToNamespacesMap` would do anything here, as `this.#findOptions` are the input options from the `createPointInTimeFinder` call, and those are not modified / enhanced by the wrappers.  

This is the call to `this.#client.openPointInTimeForType` that will effectively go though the modification performed in the security wrapper to build the map.